tests/views/test_message.py

```diff
@@ -1612,6 +1615,128 @@ def test_conversation_reply_no_edit_permission(self, mock_request, mock_get_jwt)
         self.assertNotIn("Close conversation".encode(), response.data)
         self.assertNotIn("Send message".encode(), response.data)
 
+    @requests_mock.mock()
```

Comment:
Not a fan of these tests, sorry. Do we have to use the client? because it really is causing a lot of mocking, maybe we are do but it feels a lot of boiler plate between the tests (duplication), can it not be done cleaner?  I also don't really understand the difference between test_flash_redirect_to_technical_inbox and test_return_to_technical_inbox

Reply:
This has been improved along with the other tests.

The reason for the split in tests is that one is using the return to survey link whilst the other is being redirected via the send message action. I've updated the names of the tests to better reflect this.